traffic_ops/traffic_ops_golang/deliveryservice/servers/servers.go

```diff
@@ -191,9 +192,6 @@ func (dss *TODeliveryServiceServer) readDSS(h http.Header, tx *sqlx.Tx, user *au
 		}
 		offset *= limit
 	}
-	if orderby == "" {
-		orderby = "deliveryService"
-	}
```

Comment:
Does this really result in significant performance gains? Because it makes the API less "functional", in the sense that two instances with the same data might return different responses for the same request. Or the same instance with unchanged data might return different responses for identical requests - though I think that's less likely.

Reply:
It definitely makes a significant gain in `cost` as determined by `EXPLAIN ANALYZE ...`. Just selecting the `server` and `deliveryService` columns, I get `4311.89` _without_ ordering and `8031.62` _with_ ordering. That's about a 46% reduction in cost.

That said, I was kind of on the fence about removing this default ordering vs giving the client the ability to prevent any ordering -- e.g. `?orderby` without a value. IIRC the existing code would still give that a default ordering since it doesn't check if the param actually exists in the request URL. So I would either remove the default ordering or make it check if the param exists in the URL. If the param exists without a value, it wouldn't give it a default. What do you think of that?

Reply:
Done.